flink-core/src/main/java/org/apache/flink/api/common/cache/DistributedCache.java

```diff
@@ -79,6 +80,28 @@ public DistributedCacheEntry(String filePath, Boolean isExecutable, byte[] blobK
 			this(filePath, isExecutable, blobKey, false);
 		}
 
+		@Override
```

Comment:
I must be missing something but are these methods relevant to this PR?

Reply:
That's not mandatory. I could've implemented it in tests in an asserter. I think it does not harm to have a proper `equals/hashcode`